src/api/EscargotPublic.cpp

```diff
@@ -1194,12 +1194,12 @@ void DebuggerOperationsRef::BreakpointOperations::getStackTrace(DebuggerOperatio
     outStackTrace.clear();
 
     for (uint32_t i = 0; i < size; i++) {
-        if ((size_t)stackTraceDataVector[i].second.loc.actualCodeBlock != SIZE_MAX) {
-            ByteCodeBlock* byteCodeBlock = stackTraceDataVector[i].second.loc.actualCodeBlock;
+        if ((size_t)stackTraceDataVector[i].loc.actualCodeBlock != SIZE_MAX) {
```

Comment:
if you don't mind, could you replace c-style casting with c++ style casting? 
(It is not required, so you can ignore it :smiley:)

Reply:
Sure, I'll fix it

Reply:
Thank you for hard work!